Scripts/update-translations.rb

```diff
@@ -10,6 +10,7 @@
 # * German
 # * Greek
 # * English
+# * English (Canada eh)
```

Comment:
I'm not sure it's actually used for anything, but do you want to add `en-ca` to line 5 too?